### ISSUE TITLE
docs: how to revive an expired IBC client

### ISSUE THREAD

**Issue by reecepbcups** [cosmos/relayer#1475]
_Request from the interchain discord (parveenkaushik)_

We should have a guide similar to https://medium.com/simplystaking/reviving-an-ibc-client-with-governance-hermes-relayer-362a1da4814d

https://github.com/cosmos/relayer/pull/876 may be an initial reference to help automate this.

Would be nice to auto generate the submit proposal for sdk v45 and 46+ chains. This way 1 command could handle the entire flow to revive for better UX? _(just a thought)_

**Comment by TheMarstonConnell** [comment]
The 1 command feature would be incredible

**Comment by kpettijohn** [comment]
I agree this would be super helpful. I'm attempting to revive an expired IBC client now and I can't seem to find any good docs around it at the moment.
